fmriprep/__about__.py

```diff
@@ -108,6 +108,8 @@
 ]
 
 LINKS_REQUIRES = [
+    'git+https://github.com/me-ica/tedana.git@'
+    'cb01d725d05f0102e2da89ad50c32905c68a79fd#egg=tedana',
```

Comment:
Since we're merging to `multiecho` and not `master`, this isn't urgent, but it would be good to depend on a release version of tedana before we have fMRIPrep releases that depend on this commit.

Reply:
:+1: yes, there's a growing list of things `tedana` needs, but cutting a release soon is definitely one of them